ssr/src/page/post_view/mod.rs

```diff
@@ -176,7 +176,7 @@ pub fn PostViewWithUpdates(initial_post: Option<PostDetails>) -> impl IntoView {
         video_queue.update_untracked(|v| {
             if v.len() > 1 {
                 // Safe to do a GC here
-                let rem = 0..(current_idx.get_untracked().saturating_sub(3));
+                let rem = 0..(current_idx.get_untracked().saturating_sub(5));
```

Comment:
reason for this ?

Reply:
If we navigate back to the home page, keeping last 6 videos and removing the previous videos from the queue and updating the current index to 5